wolfcrypt/src/pkcs7.c

```diff
@@ -3502,17 +3502,20 @@ static int wc_PKCS7_VerifyContentMessageDigest(PKCS7* pkcs7,
 
         digestBuf = digest;
         digestSz = wc_HashGetDigestSize(hashType);
-
+        if (digestSz < 0) {
+            WOLFSSL_MSG("Invalid hash type");
+            return digestSz;
```

Comment:
This will leak `digest` for `WOLFSSL_SMALL_STACK`. Cleanup like line 3498 above is required.

Reply:
Check